src/state/ContextProvider.tsx

```diff
@@ -41,7 +41,7 @@ const initialContext: ContextType = {
     add: () => null,
     remove: () => null
   },
-  theme: '',
+  theme: 'Default',
```

Comment:
same with the lowercase `default` here

we should also change the fallback to `default` in GlobalState.tsx:115